pkg/cluster/config/encoding/scheme.go

```diff
@@ -43,59 +49,172 @@ func init() {
 func AddToScheme(scheme *runtime.Scheme) {
 	utilruntime.Must(config.AddToScheme(scheme))
 	utilruntime.Must(v1alpha1.AddToScheme(scheme))
-	utilruntime.Must(scheme.SetVersionPriority(v1alpha1.SchemeGroupVersion))
+	utilruntime.Must(v1alpha2.AddToScheme(scheme))
+	utilruntime.Must(scheme.SetVersionPriority(v1alpha2.SchemeGroupVersion))
+}
+
+// Load reads the file at path and attempts to convert into a `kind` Config; the file
+// can be one of the different API versions defined in scheme.
+// If path == "" then the default config is returned
+func Load(path string) (*config.Config, error) {
+	if path == "" {
```

Comment:
are there messages in the pipeline saying "using default config path" vs "using **your** config path"

Reply:
The only use case where "your config path" will be used, is when the user pass `--config` explicitly, so he/she is already aware of what's going on...